test/IntegrationTests/BuildTests.DistributionStructure_alpine-linux.verified.txt

```diff
@@ -45,50 +46,72 @@
+  /store/x86/net6.0/microsoft.extensions.configuration.abstractions/7.0.0/lib/net8.0/Microsoft.Extensions.Configuration.Abstractions.dll,
+  /store/x86/net6.0/microsoft.extensions.configuration.binder/7.0.0/lib/net8.0/Microsoft.Extensions.Configuration.Binder.dll,
+  /store/x86/net6.0/microsoft.extensions.configuration/7.0.0/lib/net8.0/Microsoft.Extensions.Configuration.dll,
+  /store/x86/net6.0/microsoft.extensions.dependencyinjection.abstractions/7.0.0/lib/net8.0/Microsoft.Extensions.DependencyInjection.Abstractions.dll,
+  /store/x86/net6.0/microsoft.extensions.dependencyinjection/7.0.0/lib/net8.0/Microsoft.Extensions.DependencyInjection.dll,
+  /store/x86/net6.0/microsoft.extensions.logging.abstractions/7.0.1/lib/net8.0/Microsoft.Extensions.Logging.Abstractions.dll,
+  /store/x86/net6.0/microsoft.extensions.logging.configuration/7.0.0/lib/net8.0/Microsoft.Extensions.Logging.Configuration.dll,
+  /store/x86/net6.0/microsoft.extensions.logging/7.0.0/lib/net8.0/Microsoft.Extensions.Logging.dll,
+  /store/x86/net6.0/microsoft.extensions.options.configurationextensions/7.0.0/lib/net8.0/Microsoft.Extensions.Options.ConfigurationExtensions.dll,
+  /store/x86/net6.0/microsoft.extensions.options/7.0.0/lib/net8.0/Microsoft.Extensions.Options.dll,
+  /store/x86/net6.0/microsoft.extensions.primitives/7.0.0/lib/net8.0/Microsoft.Extensions.Primitives.dll,
+  /store/x86/net6.0/system.diagnostics.diagnosticsource/7.0.2/lib/net8.0/System.Diagnostics.DiagnosticSource.dll,
+  /store/x86/net7.0/microsoft.extensions.configuration.abstractions/7.0.0/lib/net8.0/Microsoft.Extensions.Configuration.Abstractions.dll,
+  /store/x86/net7.0/microsoft.extensions.configuration.binder/7.0.0/lib/net8.0/Microsoft.Extensions.Configuration.Binder.dll,
+  /store/x86/net7.0/microsoft.extensions.configuration/7.0.0/lib/net8.0/Microsoft.Extensions.Configuration.dll,
+  /store/x86/net7.0/microsoft.extensions.dependencyinjection.abstractions/7.0.0/lib/net8.0/Microsoft.Extensions.DependencyInjection.Abstractions.dll,
+  /store/x86/net7.0/microsoft.extensions.dependencyinjection/7.0.0/lib/net8.0/Microsoft.Extensions.DependencyInjection.dll,
+  /store/x86/net7.0/microsoft.extensions.logging.abstractions/7.0.1/lib/net8.0/Microsoft.Extensions.Logging.Abstractions.dll,
+  /store/x86/net7.0/microsoft.extensions.logging.configuration/7.0.0/lib/net8.0/Microsoft.Extensions.Logging.Configuration.dll,
+  /store/x86/net7.0/microsoft.extensions.logging/7.0.0/lib/net8.0/Microsoft.Extensions.Logging.dll,
+  /store/x86/net7.0/microsoft.extensions.options.configurationextensions/7.0.0/lib/net8.0/Microsoft.Extensions.Options.ConfigurationExtensions.dll,
+  /store/x86/net7.0/microsoft.extensions.options/7.0.0/lib/net8.0/Microsoft.Extensions.Options.dll,
+  /store/x86/net7.0/microsoft.extensions.primitives/7.0.0/lib/net8.0/Microsoft.Extensions.Primitives.dll,
+  /store/x86/net8.0/microsoft.extensions.configuration.abstractions/7.0.0/lib/net8.0/Microsoft.Extensions.Configuration.Abstractions.dll,
```

Comment:
Folder structure should be something like below and we need to use 8.0 preview version of Microsoft.Extensions.Configuration.Abstractions package.
`/store/x64/net8.0/microsoft.extensions.configuration.abstractions/8.0.0-rc.1.23419.4/lib/net8.0/Microsoft.Extensions.Configuration.Abstractions.dll`

Reply:
The current version structure is working fine. References are taken probably by Additionsal Store definitions